search-app/resources/smart-handoff/giovanni-schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "@context": "http://schema.org",
+  "@type": "ServiceChannel",
+  "url": "https://giovanni.gsfc.nasa.gov/giovanni",
+  "providesService": {
+    "@type": "Service",
+    "name": "Giovanni",
+    "url": "https://giovanni.gsfc.nasa.gov/giovanni",
+    "potentialAction": {
+      "@type": "SearchAction",
+      "target": "https://giovanni.gsfc.nasa.gov/giovanni/#service=TmAvMp&starttime={start}&endtime={end}&bbox={box}&dataKeyword={collection}&data={variables}",
+      "query-input": [
+        {
+          "@type": "PropertyValueSpecification",
+          "valueRequired": false,
+          "valueName": "start",
+          "defaultValue": {
+            "@id": "schema:startDate",
+            "@type": "Property"
+          }
+        },
+        {
+          "@type": "PropertyValueSpecification",
+          "valueRequired": false,
+          "valueName": "end",
+          "defaultValue": {
+            "@id": "schema:endDate",
+            "@type": "Property"
+          }
+        },
+        {
+          "@type": "PropertyValueSpecification",
+          "valueRequired": false,
+          "valueName": "box",
+          "defaultValue": {
+            "@type": "Place",
+            "geo": {
+              "@type": "GeoShape",
```

Comment:
Somewhere in this description we have to describe how to unambiguously formulate the parameter value in the template. Does this parameter do that? If so, it should be box right? I don't think the potentialAction supports that yet and we have to add it. This is what I meant by my ValuteType comments in the slack channel

Reply:
This ticket is just to provide the schemas as defined. As to how to draft the schema or make improvement. It is out of scope for this ticket.

Reply:
This is a vital part of the schema though. A client needs to know how the application accepts things like bounding box etc. Is there a ticket for that?

Reply:
I assume this is covered by the ticket that will actually use the schema. i.e. EDSC-3222.